dependencies.yaml

```diff
@@ -759,6 +796,31 @@ dependencies:
         packages:
           - dask-cuda==24.10.*,>=0.0.0a0
           - *numba
+  depends_on_libcudf:
+    common:
+      - output_types: conda
+        packages:
+          - &libcudf_unsuffixed libcudf==24.10.*,>=0.0.0a0
+      - output_types: requirements
+        packages:
+          # pip recognizes the index as a global option for the requirements.txt file
+          # This index is needed for rmm, cubinlinker, ptxcompiler.
```

Comment:
```suggestion
          # This index is needed for libcudf-cu{11,12}.
```

Reply:
ugh thank you, sorry for the carelessness.